WordPress/gutenberg
# Reinstate correct cover block attributes for deprecations

## Description
Fixes #15441

The incorrect attributes were specified in the `deprecated.js` file for the cover block. It looks like the attributes for the button block were accidentally pasted into the file instead of the attributes for the cover block.

## How has this been tested?
1. Roll back to a past version of Gutenberg (I used 5.2)
2. Created a new post with a cover block and save it
3. Roll forwards to the latest version of Gutenberg
4. Open up the post created in step 2.
5. Observe that the cover block fails validation
6. Checkout this branch and reload the post
7. Observe that the block works again.

## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
